setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(name='gym_mupen64plus',
       version='0.0.3',
-      install_requires=['gym==0.7.4',
+      install_requires=['gym==0.14.0',
```

Comment:
Should this also be `gym==0.15.4`?

```suggestion
      install_requires=['gym==0.15.4',
```